bootstrap.sh

```diff
@@ -110,7 +110,7 @@ done
 
 # Symlink /opt/genestack/kustomize.sh to
 # /etc/genestack/kustomize/kustomize.sh
-ln -s $base_source_dir/base-kustomize/kustomize.sh $base_target_dir/kustomize/kustomize.sh
+ln -s $base_source_dir/base-kustomize/kustomize.sh $base_target_dir/kustomize.sh
```

Comment:
I think this needs to be 
```suggestion
ln -s $base_source_dir/kustomize.sh $base_target_dir/kustomize.sh
```

Reply:
without that, we get 
``` shell
stat /etc/genestack/kustomize/kustomize.sh
  File: /etc/genestack/kustomize/kustomize.sh -> /opt/genestack/base-kustomize/base-kustomize/kustomize.sh
  Size: 57        	Blocks: 0          IO Block: 4096   symbolic link
Device: 254,10	Inode: 805593618   Links: 1
Access: (0777/lrwxrwxrwx)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2024-11-22 00:20:30.041667052 -0600
Modify: 2024-11-22 00:20:29.985666287 -0600
Change: 2024-11-22 00:20:29.985666287 -0600
 Birth: 2024-11-22 00:20:29.985666287 -0600
```

Reply:
does it make sense to change this command to use the force flag?

Reply:
I changed the one line in bootstrap.sh because it was obvious, but that file has many more issues. It needs to be looked at in more detail. It is basically broken. That was not the original scope of this pr so I did not dig in deep